TEMPLATES/.bandit.yml

```diff
@@ -86,6 +86,10 @@ tests:
 
 # (optional) list skipped test IDs here, eg '[B101, B406]':
 skips:
+  - B101
+  - B301
+  - B302
```

Comment:
I think B101 (assert_used) and B311(random)  should be skiped, however 301, 302 and 403 findings might still be worth double-checking.

Reply:
Discussed offline; 301 (alerts on every usage of `pickle()`) and 403 (on importing `pickle`) are somewhat redundant. We will skip 301  as that's the spammier of the two, but keep 403 as that will still make developers pause -- just once -- and think about the where the data's coming from.